charts/velero/templates/deployment.yaml

```diff
@@ -139,6 +139,10 @@ spec:
       nodeSelector:
         {{- toYaml . | nindent 8 }}
     {{- end }}
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 6 }}
```

Comment:
curious why this one is `nindent 6` whereas the fields just above and below are `nindent 8`?

Reply:
Is `restic-daemonset` also need this when users choose to deploy restic?

Reply:
@skriss Nothing big, just personal preference of yaml list indentation
I prefer this;
```
imagePullSecrets:
- name: foo.secret.name
```

over this
```
imagePullSecrets:
  - name: foo.secret.name
```

But not a huge deal. Sorry the delayed response. I have updated `nindent 6` to `nindent 8` to stick with consistency